source/_patterns/02-organisms/content-headers/content-header-journal.mustache

```diff
@@ -1,136 +1,159 @@
-            {{#subjects}}
-              <li class="content-header__subject_list_item">
-                <a href="{{url}}" class="content-header__subject_link">
-                  <span class="content-header__subject">{{{name}}}</span>
-                </a>
-              </li>
-            {{/subjects}}
-          </ol>
-        {{/hasSubjects}}
-      {{/header}}
-    </div>
-    <div class="content-header-grid__main">
-
-      <div class="content-header__body">
-        <h1 class="content-header__title {{#titleLength}}content-header__title--{{titleLength}}{{/titleLength}}{{^titleLength}}content-header__title--medium{{/titleLength}}">{{{title}}}</h1>
-
-        {{#impactStatement}}
-          <div class="content-header__impact-statement">{{{impactStatement}}}</div>
-        {{/impactStatement}}
-
-      </div>
-
-      {{#authors}}
-        {{> molecules-authors}}
-      {{/authors}}
-
-      <div class="content-header__footer">
-
-        {{#isOneColumn}}
-          <div class="content-header__footer-column-left">
-          {{#contextualData}}
-            {{> molecules-contextual-data }}
-          {{/contextualData}}
-        {{/isOneColumn}}
-
-        {{#meta}}
-          <div class="content-header__meta">
-            {{> molecules-meta-journal}}
-          </div>
-        {{/meta}}
-
-        {{#doi}}
-          {{> atoms-doi}}
-        {{/doi}}
-
-      {{#licence}}
-        <ul class="content-header__icons">
-          <li>
-            <a href="https://en.wikipedia.org/wiki/Open_access"
-                class="content-header__icon content-header__icon--oa">
-                  <span class="visuallyhidden">Open access</span>
-            </a>
-          </li>
-          <li>
-            <a href="{{licence}}" class="content-header__icon content-header__icon--cc">
-              <span class="visuallyhidden">Copyright information</span>
-            </a>
-          </li>
-        </ul>
-      {{/licence}}
-      {{#isOneColumn}}
+        class="content-header content-header-journal wrapper{{#isMagazine}} page--magazine{{/isMagazine}}{{#header.possible}} content-header--header{{/header.possible}}{{#socialMediaSharers}} content-header--has-social-media-sharers{{/socialMediaSharers}}"
+        data-behaviour="ContentHeader">
+
+    <div class="content-container-grid{{#isOneColumn}} content-header__one-column{{/isOneColumn}}">
+        <div class="content-header-grid-top">
+
+            {{#breadcrumb}}
+                {{> atoms-breadcrumb}}
+            {{/breadcrumb}}
+
+            {{#header}}
+
+                {{#hasSubjects}}
+                    <ol class="content-header__subject_list">
+                        {{#subjects}}
+                            <li class="content-header__subject_list_item">
+                                {{#url}}
+                                    <a href="{{url}}" class="content-header__subject_link">
+                                        <span class="content-header__subject">{{{name}}}</span>
```

Comment:
span does not need to be here as styling can be done in content-header__subject_link.

Reply:
I think this is not a requirement for this ticket. It appears as a change just because Alex copied all the content from another file to this one. This ticket is about curation labels. The span is about MSA labels.